addons/website_blog/views/snippets/snippets.xml

```diff
@@ -9,7 +9,7 @@
 
 <template id="snippet_options" inherit_id="website.snippet_options" name="Blog snippet options">
     <xpath expr="." position="inside">
-        <div data-js="BlogPostTagSelection" data-selector=".o_wblog_post_page_cover" data-target="#o_wblog_post_name">
+        <div data-js="BlogPostTagSelection" data-selector=".o_wblog_post_page_cover" data-target="#o_wblog_post_name" data-no-check="true">
```

Comment:
@bso-odoo Actually I rested this while adapting for master: this does not work for "regular" cover, you cannot update the tags. Normal since here we search for o_wblog_post_name inside the cover which is not the case with "regular". Would create a new PR in stable to fix that ? (or create an extra task for later about it, as you prefer).

Meanwhile, I'll forward-port this one as it is :+1: